authors/apps/authentication/tests/test_validation.py

```diff
@@ -0,0 +1,93 @@
+from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from . import BaseTest
+
+
+class ValidationTests(TestCase, BaseTest):
+
+    def setUp(self):
+        self.client = APIClient()
+        self.user = {"user": {"username": "llll",
+                              "email": "l@gmail.com",
+                              "password": "p1234567"
+                              }
+                     }
```

Comment:
@cmplx-xyttmt let's define these in the `BaseTest` or use the ones already created there.

Reply:
@arthuroe , thanks for pointing that out. I've done that.